linux_admin.gemspec

```diff
@@ -38,7 +37,7 @@ registration, updates, etc.
   spec.add_dependency "awesome_spawn",        "~> 1.3"
   spec.add_dependency "inifile"
   spec.add_dependency "more_core_extensions", "~> 3.0"
-  spec.add_dependency "nokogiri",             ">= 1.8.2", "~> 1.8"
+  spec.add_dependency "nokogiri",             ">= 1.10.3", "~> 1.10"
```

Comment:
Warning, I have no looked at hakiri but @Fryguy schooled me in the ways of the ! operand with rubygems versions, in case you need to exclude specific old patch releases of a prior minor version while also wanting newer versions:

https://github.com/ManageIQ/manageiq-schema/commit/a45bd687499e1b4a76e57278aa6d01a6ff44f32f

Reply:
Okay, I'll take a look to see if I can do something similar to maintain compatibility with gems-pending and also fix hakiri here.